spec/configuration.yml.example

```diff
@@ -7,11 +7,11 @@ root:
 user:
   host: localhost
   username: LOCALUSERNAME
-  password:
+  password: test
```

Comment:
Updated to drop the unused toplevel db configs. The numeric-db spec should just create the numeric db using the main test db creds. Much simpler!